### PR TITLE
#166980200 Setup redux

### PR DESCRIPTION
**Pull request by wangonya**
#### What does this PR do?
- Sets up redux
#### Description of Task to be completed?
- install redux dependencies
- setup root reducer
- setup store
#### How should this be manually tested?
* Pull the branch:
    `git pull origin ch-setup-redux-166980200`
* Install dependencies:
    `yarn`
* Run local server:
    `yarn develop`
#### What are the relevant pivotal tracker stories?
[#166980200](https://www.pivotaltracker.com/story/show/166980200)
#### Screenshots (if appropriate)
Screenshot 2019-07-02 at 09 51 24

### CODE REVIEW

**Review by willeswa — APPROVED** [review]

**Review by jmusila — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
+    "react-dom": "^16.8.6",
+    "react-redux": "^7.1.0",
+    "redux": "^4.0.1",
+    "redux-thunk": "^2.3.0"
   },
   "scripts": {
     "develop": "webpack-dev-server",
-    "build": "webpack --mode production"
+    "build": "webpack --mode production",
+    "coveralls": "jest --detectOpenHandles --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
```
Comment (wekesa931) [inline package.json]: What does this script do?

Comment (wangonya) [reply package.json]: @kburudi requested that I should add it for coverage to be recorded. His branch doesn't have a `package.json`

**Review by ceexon — APPROVED** [review]

**Comment by FlevianK** [comment]
LGTM

**Pull request merged**